pkg/importer/importer.go

```diff
@@ -342,6 +348,12 @@ type RunContext struct {
 	lastProgress *ProgressMessage
 }
 
+// UsePaidAPI returns `true` if the run is going to use the paid API(s) when retrieving data.
```

Comment:
The typical Go way to document boolean functions is "reports whether" instead of "returns true if"